astrokat/observe_main.py

```diff
@@ -237,6 +237,7 @@ def __init__(self, opts, correlator=None):
 
     def __enter__(self):
         # Verify subarray setup correct for observation before doing any work
+        user_logger.info('Running astrokat version - {}'.format(astrokat.__version__))
```

Comment:
We tend to stick to the old `%-style of string formatting` with logging and append the variable data as arguments.
But this also works. 

e. g
`        user_logger.info('Running astrokat version - %s', astrokat.__version__)`

Reply:
+1 for `%` formatting of log messages.